pkg/apis/batch/validation/validation_test.go

```diff
@@ -1283,6 +1283,114 @@ func TestValidateJobUpdate(t *testing.T) {
 				AllowMutableSchedulingDirectives: true,
 			},
 		},
+		"update completions and parallelism to same value is valid": {
```

Comment:
I would expect to see a feature gate loop in this code since you are using that feature gate in validation.go.  

Ideally you would want to turn the FeatureGate for your code and then run these tests and also run it with the feature gate off.

Reply:
Yea I didn't see anything about disabling or enabling the feature gate in these test cases so I was wondering if it was assuming default behavior.  I guess the gate is true but we should maybe test that the gate is false in case of rollback?

Reply:
yes, it's true because it will be beta. We need to have coverage for when the feature is disabled.